packages/bitcoin/test/BitcoinSigner.test.ts

```diff
@@ -0,0 +1,131 @@
+/* eslint-disable unicorn/no-useless-undefined, no-magic-numbers, no-loop-func, @typescript-eslint/no-non-null-assertion, unicorn/consistent-function-scoping, @typescript-eslint/no-explicit-any, @typescript-eslint/no-var-requires, camelcase */
+import { BitcoinSigner, signTx } from '../src/wallet/lib/wallet/BitcoinSigner';
+import { AddressType, ChainType, KeyPair, Network, UnsignedTransaction } from '../src/wallet/lib/common';
```

Comment:
Maybe we can combine the imports? 🤔 

```suggestion
import { AddressType, ChainType, KeyPair, Network, UnsignedTransaction, BitcoinSigner, signTx } from '../src/wallet';
```

Reply:
done here https://github.com/input-output-hk/lace/pull/1855/commits/d501a183e05621c6cf939486b60caf9e2ad2f1a0